### ISSUE TITLE
`elfeed-show-next` results in an error when using a popup to show the post

### ISSUE THREAD

**Issue by brabalan** [hlissner/doom-emacs#646]
When I do `elfeed-show-next` (bound to g j) I get a
```
window-normalize-window: #<window 60> is not a valid window
```
(with the number always increasing)

If I set this
``` emacs-lisp
(set! :popup "^\\*elfeed-entry" :ignore)
```
the problem goes away.

To fix it, I suggest changing `+rss/delete-pane` as follows:
``` emacs-lisp
(defun +rss/delete-pane ()
  "Delete the *elfeed-entry* split pane."
  (interactive)
  (let* ((buff (get-buffer "*elfeed-entry*"))
         (window (get-buffer-window buff)))
    (kill-buffer buff)
    (+popup/close window)
    ;;(delete-window window)
    ))
```

**Comment by hlissner** [comment]
This should be fixed in 8f84768. Please update and let me know if you have any trouble.

**Comment by brabalan** [comment]
It's working now, thanks!
